pkg/demoinfocs/stringtables.go

```diff
@@ -308,6 +310,16 @@ func parseStringTable(
 		return items
 	}
 
+	defer func() {
+		err := recover()
+		if err != nil {
+			p.eventDispatcher.Dispatch(events.ParserWarn{
```

Comment:
could we re-try parsing with the previous logic here? or is there no point?

Reply:
it always leads to empty items